megatron/core/inference/contexts/dynamic_context.py

```diff
@@ -384,7 +384,7 @@ def __init__(
 
         # Set max_total_requests, max_active_requests, max_tokens.
         self.max_total_requests = self.block_allocator.total_count - 1  # -1 for dummy block
-        self.max_active_requests = self.block_allocator.active_count
+        self.max_active_requests = math.floor(self.block_allocator.active_count / tp_size) * tp_size
```

Comment:
@lmcafee-nvidia Is this a good place to make the "bring back max_requests flag" change we talked about?